rfcs/RFC-0012-protocol-for-incentivization-of-exits/0012-protocol-for-incentivization-of-exits.md

```
@@ -0,0 +1,277 @@
+
+The Exit node generates the `ExitCommitmentRequest` as follows:
+
+It chooses a random scalar `b` (via a CSPRNG) and computes `ExitCommitment = b * BP` where `BP` is the point of large order on curve `C`, and stores `b` associated with the pseudonym `P`.
+
+The Exit node chooses number `t` which will be a degree of polynomials. And `m` the number of polynomials. Both SHOULD BE unsigned 16-bit numbers.
+
+It creates a 32-bit unsigned integer `params = (m << 16) || (t+1)`. In other words, `m` should be encoded as upper 16-bit half and `t+1` as lower 16-bit half of the `params` integer.
+
+The message SHOULD BE constructed as follows:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)